source/SquadLeader/.idea/encodings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure what this .idea folder is - but just wanted to check if it was a local user settings folder that shouldn't be added to source control (or conversely added to gitignore - the `"."` on the folder name is the only thing that made it suspicious)